src/components/pages/maintenance/maintenance.js

```diff
@@ -335,7 +335,7 @@ class MaintenancePage extends Component {
     let breadcrumbs;
     const pathName = this.props.location.pathname;
     const alarmsGridData = this.props.alarmsGridData;
-    const parentLink = <Link to={'/maintenance'}>{lang.MAINTENANCE}</Link>;
+    const parentLink = <Link to={'/maintenance'} onClick={() => this.setState({ contextBtns: true })}>{lang.MAINTENANCE}</Link>;
```

Comment:
Avoid creating static functions in the render method. That will create a new function instance each time render is called, causing the children to re-render when nothing actually changed.

Reply:
Sure.